### ISSUE TITLE
Segregate the code in order to allow for bigger and better development

### ISSUE THREAD

**Issue by dadoming** [dadoming/SortPeopleIntoTeams#3]
What I mean by this is:
______________________________________________
Create maybe a prompt that just needs to input like:
1. Normal Sort
2. 42 sort
3. Other sorts...
______________________________________________
After that prompt for the relative path of the names_file.txt.
______________________________________________
Prompt for the minimum number of participants per team
Prompt for the maximum number of participants per team
______________________________________________

The prompt part will basically grab and parse the values based on the sorting algorithm selected.

All of this would be separated into different files in the root of the directory,
For now I think this would be fine.

|
|_ Prompts
|_ Sorts
|       |___ 42_Sort.py
|       |___ normal_sort.py
|       |___ other_sorts.py
|
|_ main.py
|_ READ.me
|_ requirements.txt

**Comment by dadoming** [comment]
Where would the flags be inserted here?

**Comment by MariaAguiar** [comment]
Without any flags, the only sorting you have is the random function on the names.

Flags (either FLAG, RANGE or VALUES) turn their respective columns into additional sorting criteria:

- FLAG forces an equal value of 1s and 0s on all teams.
  It can be used for yes and no questions: "are you fluent in this topic?"; "have you been to this type of event before?"

- RANGE and VALUES are for columns filled with somewhat repeated categories, like 42 Ranks or occupation categories (Programmer, HR, Manager) or (Junior, Plain, Senior) , etc. Using the example of 42 Ranks:
  VALUES makes Rank 5s (for example) be equally represented in all teams as much as possible
  RANGE makes the sum of all possible Ranks in each team the same or similar for all teams
  (You are not forced to use both on the same column. It is a possibility to avoid data repetition)

Each column with a flag constitutes an additional, independent criteria to take into account, meaning you can add as many flags as you want in as many columns as you want in a single sorting to further restrict it (within realistic bounds)

In short, it is a more flexible, but controlled type of sorting... it was meant to be customisable, while keeping it good and feasible within reason...

Customized sort?

**Comment by MariaAguiar** [comment]
Additionally to this customized sort, you could do a weighted version, where you modify the order in which you loose each criteria until you get a viable solution (based on user input)
